examples/simple-genai-server/README.md

```diff
@@ -37,45 +37,17 @@ Follow the instructions for
 
 ### Set the Game Servers on the Same Cluster as the GenAI Inference Server
 
-To set the Game Servers on the same cluster as Google for Games GenAI inference server
-[create an Agones controller node pool](https://agones.dev/site/docs/installation/creating-cluster/gke/#optional-creating-a-dedicated-node-pool),
-optionally [create a metrics node pool](https://agones.dev/site/docs/installation/creating-cluster/gke/#optional-creating-a-metrics-node-pool),
-create a node pool for your game servers:
-```bash
-gcloud container node-pools create game-servers  \
-  --cluster=[CLUSTER_NAME] \
-  --zone=[COMPUTE_ZONE] \
-  --tags=game-server \
-  --node-taints agones.dev/role=gameserver:NoExecute \
-  --node-labels agones.dev/role=gameserver \
-  --num-nodes=4 \
-  --machine-type=e2-standard-4
-```
-and create a firewall in the same VPC network as your inference server to access the game servers:
-```bash
-gcloud compute firewall-rules create gke-genai-quickstart-game-server-firewall-tcp \
-  --description "Firewall to allow game server tcp traffic" \
-  --allow tcp:7000-8000 \
-  --target-tags game-server \
-  --network vpc-genai-quickstart \
-  --priority 998
-```
-Note that if you use this dedicated game-servers node-pool you will also need to use the `toleration`
-and `nodeSelector` in the [gameserver_manualchat.yaml](gameserver_manualchat.yaml) or
-[gameserver_autochat.yaml](gameserver_autochat.yaml).
-
-Lastly, follow the instructions for
-[installing Agones](https://agones.dev/site/docs/installation/install-agones/).
+To set the Game Servers on the same cluster as Google for Games GenAI inference server follow the instructions for [installing Agones](https://agones.dev/site/docs/installation/install-agones/).
 
 ## Setting up the Game Server
```

Comment:
Suggest that you change section heading to "How to modify the inference request structure" and just keep the paragraph "If you need a different request structure,..."